src/__tests__/events.js

```diff
@@ -128,6 +128,9 @@ const eventTypes = [
   },
 ]
 
+// native select event isn't passing
+const nonNativeEvents = {doubleClick: 'dblclick', select: true}
```

Comment:
Should `doubleclick` even be in the events list?

Also, `select` probably doesn't work because JSDOM, which is unfortunate.

I probably wouldn't implement things this way and instead I would just have if statements in the test itself for each of these individually with comments explaining why these are special cases. No reason to have this indirection/abstraction.

Reply:
@kentcdodds I made some changes based on your feedback.
>instead I would just have if statements in the test itself for each of these individually with comments explaining why these are special cases
Done
> Should doubleclick even be in the events list?
I think it makes sense to test both the doubleClick (synthetic) and dblclick (native) events. 
> Also, select probably doesn't work because JSDOM, which is unfortunate.
I included a fix for this in the latest commit.